README.md

```diff
@@ -58,41 +58,41 @@ Here's a typical controller:
 ```javascript
 // user.js
 
-const util   = require(__dirname + '/../helpers/util'),
-const mysql  = require('anytv-node-mysql'),
-const moment = require('moment');
+const config    = require(__dirname + '/../config/config');
+const mysql     = require('anytv-node-mysql');
+const moment    = require('moment');
 
 
 
 exports.update_user = (req, res, next) => {
 
-    const data = util.get_data(
-        {
-            user_id: '',
-            _first_name: '',
-            _last_name: ''
-        },
-        req.body
-    );
-
-
     function start () {
-        let id;
-
-        if (data instanceof Error) {
-            return res.warn(400, {message: data.message});
-        }
 
-        id = data.user_id;
-        delete data.user_id;
-
-        mysql.use('my_db')
-            .query(
-                'UPDATE users SET ? WHERE user_id = ? LIMIT 1;',
-                [data, id],
-                send_response
-            )
-            .end();
+         req.checkBody({
+            user_id: 'required|alpha_dash',
+            first_name: 'string',
+            last_name: 'string'
+         });
+
+        req.getValidationResult().then(result => {
```

Comment:
this is very weird. a `then` function that doesn't follow the error first callback. why did not just return this result on req.checkBody? o.O